jvb/src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -1119,6 +1126,24 @@ public void endpointMessageTransportConnected(@NotNull AbstractEndpoint abstract
         }
     }
 
+    public AbstractEndpoint getEndpointBySsrc(long ssrc)
+    {
+        return endpointsBySsrc.get(ssrc);
+    }
+
+    public void addEndpointSsrc(@NotNull AbstractEndpoint endpoint, long ssrc)
+    {
+        AbstractEndpoint oldEndpoint = endpointsBySsrc.put(ssrc, endpoint);
+        if (oldEndpoint != null && oldEndpoint != endpoint) {
+            logger.warn("SSRC " + ssrc + " moved from ep " + oldEndpoint.getId() + " to ep " + endpoint.getId());
+        }
+    }
+
+    public void removeEndpointSsrc(@NotNull AbstractEndpoint endpoint, long ssrc)
```

Comment:
Probably an artifact of this being a WIP, but will need to make sure to remove all references to an endpoint from this map when the endpoint leaves so there's no leak.

Reply:
It's removed in `endpointExpired`, is there some case you think it's missing?

Reply:
~Oh, it is?  I searched for `removeEndpointSsrc` in the diff and only saw it called in `setReceiveSsrcs`~. Ah, I see `endpointExpired` removes from the map directly.